shared/common-adapters/emoji.desktop.js

```diff
@@ -12,7 +12,7 @@ const backgroundImageFn = (set: string, sheetSize: string) => emojiSet
 
 // Size 0 is cause we want the native emoji for copy/paste and not for rendering
 const EmojiWrapper = (props: Props) => {
-  const emojiText = `:${props.children}:`
+  const emojiText = String(props.children)
```

Comment:
For `:smile:`, the parser used to output `smile`. I'm changing this so we use the colon syntax everywhere, like `:smile:`. We use the colon syntax in our components for rendering, so now we don't have to add the colons back.